### DISCUSSION TITLE
Fermata not working on AA 7.8 and 8

### DISCUSSION THREAD

**Discussion by Jotay2k** [AndreyPavlenko/Fermata#208] [Announcements]
Fermata gives black screen Up to AA 7.7
ONLY 7.7 runs. Not 7.8 nor 8.
Clean instalo with adb push from cmd as always and nothing on 7.8/8.

Any idea.

The apks were previously cloned in order to cheat AA recognition.

**Comment by AndreyPavlenko** [comment]
Have you tried installing with KingInstaller?

**Comment by Jotay2k** [comment]
I always install It cloned (Lucky Patcher) with ADB throuht CMD (PC).
Kingstaller does not work with me.
Enviado desde mi Huawei

**Comment by Hurrikhang** [comment]
https://youtu.be/x9LzDF7D3Ko
Thank me later

**Comment by Jotay2k** [comment]
KINGINSTALLER DOES NOT WOKRS ON MY HUAWEI

**Comment by Jotay2k** [comment]
Inside AA i see fermata and cartube but when i tap on them the car displays black.

**Comment by miikew** [comment]
Nord 2 - not working with LEAF 2019 Headunit

**Comment by Jotay2k** [comment]
Anyone works? Huawei P10 lite + AA 8?

**Comment by fcaronte** [comment]
> Huawei P10 lite

i think you need an android system version >9 to have it working or will never work

**Comment by AndreyPavlenko** [comment]
Try this - https://github.com/AndreyPavlenko/Fermata/discussions/122#discussioncomment-4307775
